corehq/tabs/tabclasses.py

```diff
@@ -1539,6 +1537,17 @@ def sidebar_items(self):
         if locations_menu:
             items.append((_('Organization'), locations_menu))
 
+        if (
+                user_can_view_reports(self.project, self.couch_user)
+                and has_privilege(self._request, privileges.PROJECT_ACCESS)
```

Comment:
taken from 
https://github.com/dimagi/commcare-hq/blob/81784308f125d6b531ddfd8949dc037167f2d7e3/corehq/tabs/tabclasses.py#L109-L122
to check that user has access to reports.